content/blog/2019/announcing-1.2.1/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Announcing Istio 1.2.1
+description: Istio 1.2.1 patch release.
+publishdate: 2019-07-01
+attribution: The Istio Team
+release: 1.2.1
+---
+
+We're pleased to announce the availability of Istio 1.2.1. Please see below for what's changed.
```

Comment:
do we need a blog for 1.2.1 if we want to promote 1.2.2?  It seems odd to tweet or blog about 1.2.1

Reply:
The web site UI will automatically recommend using 1.2.2. We need to have 1.2.1 present, otherwise we will have broken links from 1.2.2 pointing to 1.2.1

Reply:
I think it is required to not break the doc pipeline.